.github/workflows/ci.yml

```diff
@@ -1,17 +1,35 @@
 name: CI
 
 on:
-  push:
-    branches:
-      - main
-  pull_request:
-    branches:
-      - main
+  workflow_call:
+    inputs:
+      kind:
+        required: false
+        type: string
+        #type: choice
+        #description: Kind of build to run
+        #options:
+        #  - dev
+        #  - nigthly
+        #  - release
+        default: dev
+      package_command:
+        required: false
+        type: string
+        description: Command used to build python package
+        default: >-
```

Comment:
Will you ever need to pass a different package command?

Reply:
Yes, release vs nightly vs dev

Reply:
Ok, that's just the type of build. We can probably make this cleaner with nox...